packages/ui/src/components/shadcn/ui/dropdown-menu.tsx

```diff
@@ -79,7 +86,7 @@ const DropdownMenuItem = React.forwardRef<
   <DropdownMenuPrimitive.Item
```

Comment:
same as https://github.com/supabase/supabase/pull/17427/files#r1327465449

Reply:
Will remove it and add to all dropdown items.